custom_components/google_home/sensor.py

```diff
@@ -87,13 +89,23 @@ async def async_setup_entry(
     # Services
     platform.async_register_entity_service(
         SERVICE_DELETE_ALARM,
-        {vol.Required(SERVICE_ATTR_ALARM_ID): cv.string},  # type: ignore[dict-item]
+        {
+            vol.Required(SERVICE_ATTR_ALARM_ID): cv.string,  # type: ignore[dict-item]
+            vol.Optional(
+                SERVICE_ATTR_FORCE_REFRESH
+            ): cv.boolean,  # type: ignore[dict-item]
+        },
         GoogleHomeAlarmsSensor.async_delete_alarm,
     )
 
     platform.async_register_entity_service(
         SERVICE_DELETE_TIMER,
-        {vol.Required(SERVICE_ATTR_TIMER_ID): cv.string},  # type: ignore[dict-item]
+        {  # types: ignore[dict]
```

Comment:
This one shouldn't be needed.
```suggestion
        {
```